### ISSUE TITLE
Allow P-value cluster-forming thresholds with pseudo-t

### ISSUE THREAD

**Issue by nicholst** [nicholst/SnPM-devel#44]
Because it is such a pain, we might as well offer to convert P-values into corresponding T/F statistics based on nominal degrees of freedom when the Pseudo-t statistic is used.  Variance smoothing messes up the DF, and hence these statistics won't have the exact same interpretation as usual... however, there is no known mapping between uncorrected P-values and pseudo-t values, and hence it leaves users in a muddle.  

In FSL's randomise, we don't worry about this.  

We should change our behavior, and provide the P-to-T/F service for users, simply issuing a warning that the conversion is approximate due to unknown effective degrees of freedom.

Relevant code starts here: https://github.com/nicholst/SnPM-devel/blob/master/snpm_pp.m#L518

**Comment by cmaumet** [comment]
@nicholst: is this already implemented in https://github.com/nicholst/SnPM-devel/blob/master/snpm_pp.m#L534-L540? Or do you mean something different?

**Comment by nicholst** [comment]
Oh... oops... yes.

**Comment by nicholst** [comment]
No.. but wait... that code assumes that the statistic image is *T*... not *F*... current code won't do something sensible with F statistic.

ACTION: Need to revise code to allow sensible clustering forming threshold (from P-value) when STAT == 'F'.
